src/js/components/BreadcrumbComponent.jsx

```diff
@@ -1,7 +1,12 @@
+var _ = require("underscore");
+var classNames = require("classnames");
 var React = require("react/addons");
 var Link = require("react-router").Link;
 
 var PathUtil = require("../helpers/PathUtil");
+var StyleDimensions = require("../constants/StyleDimensions");
```

Comment:
Yeah, I kinda see the point but tbh I don't see a big added value... A cool thing to do would be to have a gulp task that parses the `variables.less` file and generates the JS equivalent.